irohad/peer_service/peer_service.cpp

```diff
@@ -14,45 +14,74 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+#include <numeric>
+#include <random>
+#include <unordered_set>
+#include <exception>
+
 #include <peer_service/peer_service.hpp>
 
-// TODO
 namespace peer_service {
 
   void initialize() {
```

Comment:
In iroha PeerService has a common state.
This has no merit of classifying.
It is not necessary to classify `ametsuchi` for the same reason. (This is a lie, there are parts requiring classification as a wrapper of DB.)